acs-apiclient/Resources/values/Colors.xml

```diff
@@ -0,0 +1,6 @@
+﻿<?xml version="1.0" encoding="UTF-8" ?>
+<resources>
+    <color name="cloudy_blue">#B7C0CC</color>
+    <color name="ash">#B7C0CC</color>
+    <color name="obsidian">#222b3c</color>
```

Comment:
💡 It might be better in this case to set the colors in code - that way they can be shared more easily between iOS and Android

Reply:
Done.

Reply:
✅